source/build/Windows/MainWin32.cpp

```diff
@@ -3,11 +3,56 @@
 #include "TestAppPch.h"
 #include "TestApp.h"
 
+#include <playfab/PlayFabApiSettings.h>
+#include <playfab/PlayFabAuthenticationContext.h>
+
 // Win32 Entry Point
 int main()
 {
-    PlayFabUnit::TestApp testApp;
+    // Settings
```

Comment:
>Settings [](start = 7, length = 8)

I don't think this block of testing belongs in main, you can move this as part of a different test where other tests are, this feels really out of place.

Reply:
The main point of this here is simplicity.
This code has to live in the exe, rather than the lib.
All the other tests live in the lib.

I think the best I could do is implement a separate test suite in the exe, containing this code.
It would look like the other tests, but it would still live in the exe.

Reply:
even though its part of exe, can it be part of testapp, I think there should be a central place where all the test logic lives

---
In reply to: [346575631](https://github.com/PlayFab/XPlatCoreTemplate/pull/46#discussion_r346575631) [](ancestors = 346575631)

Reply:
I agree.
I will address this in my timestamps branch, because it'll be my next checkin, and it is hard to go back and forth between them when they have overlapping changes.